lyrics/20-azlyrics.py

```diff
@@ -0,0 +1,17 @@
+import re
+from sys import argv
+
+import requests
+
+artist = argv[1].lower()
+title = argv[2].lower()
+artist = re.sub("[^A-Za-z0-9]+", "", artist)
+title = re.sub("[^A-Za-z0-9]+", "", title)
+
+url = "http://azlyrics.com/lyrics/" + artist + "/" + title + ".html"
```

Comment:
Do you need to escape these strings before inserting into a URL?

Reply:
No, actually. The regex makes sure only numbers and letters are in the URLs. Azlyrics has a rather simple way of fetching lyrics - there are no spaces in the artist or the title.
For example - https://www.azlyrics.com/lyrics/arianagrande/godisawoman.html